common/views/components/CheckboxRadio/CheckboxRadio.tsx

```diff
@@ -69,16 +69,18 @@ type CheckboxRadioProps = {
   name: string;
   onChange: (event: SyntheticEvent<HTMLInputElement>) => void;
   value: string;
+  ariaLabel?: string;
```

Comment:
We do pass this around a lot, we could use the `JSX.IntrinsicElements['div']` - but this is good for now.